src/hotspot/share/runtime/atomic.hpp

```diff
@@ -794,8 +794,8 @@ template<typename D, typename U, typename T>
 struct Atomic::CmpxchgImpl<
   D*, U*, T*,
   typename EnableIf<Atomic::IsPointerConvertible<T*, D*>::value &&
-                    IsSame<typename RemoveCV<D>::type,
-                           typename RemoveCV<U>::type>::value>::type>
+                    IsSame<typename std::remove_cv<D>::type,
+                           typename std::remove_cv<U>::type>::value>::type>
```

Comment:
Prefer the `_t` suffixed traits here, e.g. `typename std::remove_cv<D>::type` => `std::remove_cv_t<D>`.

Reply:
Done. Was going for the least amount of change originally.